web/html/src/components/formulas/EditGroup.tsx

```diff
@@ -0,0 +1,403 @@
+/* eslint-disable */
```

Comment:
```suggestion
```

Reply:
Hmm, looks like the migration script isn't cleaning these up correctly. I'll have to have a look later. I cleaned these up manually for now.